spec/controllers/testing_grounds/caching_spec.rb

```diff
@@ -43,10 +44,10 @@
     let(:opts) { { strategies: {} } }
 
     it 'caches the data request' do
-      get :data, id: testing_ground
+      post :data, id: testing_ground
 
       loads = network.node('CONGESTED_END_POINT_0').get(:load)
-      expect(loads.length).to eq(8760)
+      #expect(loads.length).to eq(8760)
```

Comment:
Did this request start returning something different? I'm curious why this request has become a POST, since fetching the load data should be idempotent and doesn't alter any state on the server; did that change?